ts-defs-src/test-api/action-options.d.ts

```diff
@@ -174,3 +174,42 @@ interface ResizeToFitDeviceOptions {
      */
     portraitOrientation?: boolean;
 }
+
+interface CookieOptions {
+    /**
+     * Specifies the name of the cookie.
+     */
+    name?: string;
+    /**
+     * Specifies the value of the cookie.
+     */
+    value?: string;
+    /**
+     * Specifies the domain which will be matched with the cookie.
+     */
+    domain?: string;
```

Comment:
I'd rather use URL or base URL. Because `.com` is a domain. `test.com` is a domain *name*. I'm confused TBH

Reply:
It won't be consistency to the common standard https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_where_cookies_are_sent